lib/core/src/server/config/webpack.config.default.js

```diff
@@ -19,16 +24,18 @@ export function createDefaultWebpackConfig(storybookBaseConfig) {
             },
             {
               loader: require.resolve('postcss-loader'),
-              options: {
-                ident: 'postcss', // https://webpack.js.org/guides/migrating/#complex-options
-                postcss: {},
-                plugins: () => [
-                  require('postcss-flexbugs-fixes'), // eslint-disable-line
-                  autoprefixer({
-                    flexbox: 'no-2009',
-                  }),
-                ],
-              },
+              options: postcssConfig
+                ? {
+                    postcssConfig,
```

Comment:
should it be `{ ...postcssConfig }` ?

I think will be better something like this:

```ja
postcssConfig || {
   // flexbug blah blah
}
```